CMakeLists.txt

```diff
@@ -98,6 +98,22 @@ find_package(podio REQUIRED)
 find_package(EDM4HEP REQUIRED)
 find_package(EDM4EIC 2.1 REQUIRED)
 
+# Remove PODIO_JSON_OUTPUT (ref: https://github.com/AIDASoft/podio/issues/475)
+if(TARGET EDM4HEP::edm4hep)
```

Comment:
Should not be conditional. If target is renamed, we'd like for configuration to fail loudly and allow us to adapt the workaround.

Reply:
Addressed in https://github.com/eic/EICrecon/pull/916/commits/7262ca69936a65c1c18959e7653d5a790c8b6f5c.